### PR TITLE
Some top-level README/CONTRIBUTING improvements

### PR DESCRIPTION
**Pull request by gigabo**
A few small changes to improve the landing experience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <img src="/images/reactserver_logo%402x.png" width="450px"/>
 
-# react-server monorepo [![Build Status][build-badge-img]][build-url]
+[![Build Status][build-badge-img]][build-url]
+[![NPM version][npm-badge-img]][npm-url]
```
Comment (doug-wade) [inline README.md]: What is this the version of?  Other lerna-style monorepos (https://github.com/babel/babel, https://github.com/meteor/meteor, https://github.com/emberjs/ember.js) don't do this.

Comment (gigabo) [inline README.md]: It's the main `react-server` package.  I [argued against it](https://github.com/redfin/react-server/pull/36#discussion_r54039361) at one point, too.

I'll remove it if you think it's misleading.  I just like the look of more badges. :)

Comment (doug-wade) [inline README.md]: You better prepare yourself for my inevitable "add 400 more badges" pr, then ;)

**Comment by doug-wade** [comment]
lgtm 👍

File: README.md (hunk 1 above)
Comment (gigabo) [inline README.md]: image

**Pull request merged**